recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch

```diff
@@ -0,0 +1,42 @@
+From 78a3290562bc7961d29374ab1004666562c86d72 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Wed, 22 Sep 2021 16:47:57 +0800
+Subject: [PATCH 9/9] u-boot: dts : remove usb3.0 related dts configurations
+
+This is a known issue that usb3.0 can not work under u-boot.
+The usb3.0 port can only work at usb2.0 mode. So remove the usb3.0
+related dts configurations and disable the usb3.0 serdes signal
+
+Signed-off-by: chao zeng <chao.zeng@siemens.com>
+---
+ arch/arm/dts/k3-am65-iot2050-common-pg2.dtsi | 16 +---------------
```

Comment:
Again: Patch k3-am65-iot2050-common-u-boot.dtsi or create a corresponding one for only pg2 but do not modify this one beyond includes.

Reply:
has already create a new dtsi file to restore this feature.